sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/api/Microsoft.Azure.WebJobs.Extensions.SignalRService.net5.0.cs

```diff
@@ -0,0 +1,272 @@
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    public static partial class Category
```

Comment:
Could we call it SignalRCategory and SignalREvent? Type names like Event and Category are too generic.

Reply:
They are already released classes. How about marking them as deprecated and adding new classes named SignalRCategory and SignalREvent?